lib/sms/Kconfig

```
@@ -18,6 +18,13 @@ config SMS_SUBSCRIBERS_MAX_CNT
 	help
 	  Maximum number of subscribers that can register to SMS library.
 
+config SMS_DELIVERY_REPORT
+	bool "SMS delivery reports are requested"
+	default y
+	help
+	  Request SMS delivery report by not setting TP-SRR bit in SMS header.
+	  Also, let modem handle them by using CNMI=3,2,0,0.
```

This should also be updated. When delivery reports are requested, the `CNMI` command parameters are `3,2,0,1`. This could be something like:
```suggestion
	  When delivery reports are not requested, let modem handle them by using CNMI=3,2,0,0.
```

---

Seems I've written everything like this option would disable delivery reports although it enables them.

---

This part has been reworded.